### ISSUE TITLE
Support for Ubuntu 15.04

### ISSUE THREAD

**Issue by heaven** [hw-cookbooks/postgresql#291]
```
RuntimeError
------------
Not supported release by PGDG apt repository


Cookbook Trace:
---------------
  /var/chef/cache/cookbooks/postgresql/recipes/apt_pgdg_postgresql.rb:2:in `from_file'
  /var/chef/cache/cookbooks/postgresql/recipes/client.rb:23:in `from_file'
  /var/chef/cache/cookbooks/postgresql/recipes/server.rb:20:in `from_file'

Relevant File Content:
----------------------
/var/chef/cache/cookbooks/postgresql/recipes/apt_pgdg_postgresql.rb:

  1:  if not %w(squeeze wheezy sid lucid precise saucy trusty utopic).include? node['postgresql']['pgdg']['release_apt_codename']
  2>>   raise "Not supported release by PGDG apt repository"
  3:  end
  4:
  5:  include_recipe 'apt'
  6:
  7:  file "remove deprecated Pitti PPA apt repository" do
  8:    action :delete
  9:    path "/etc/apt/sources.list.d/pitti-postgresql-ppa"
 10:  end
 11:
```

**Comment by kohenkatz** [comment]
There are no PGDG packages for 15.04 "vivid". However, there are now packages posted for 15.10 "wily", so maybe that one should be added.

**Comment by CloCkWeRX** [comment]
Maybe best reported to/asked about via https://wiki.postgresql.org/wiki/Apt#Bugs

**Comment by davejacobs** [comment]
Hi there, I think what @kohenkatz is asking is if someone would add `wily` to the list of supported PGDG distros?

**Comment by slyness** [comment]
I will work on adding 15.04 (vivid) to the next release. I will not be adding 15.10 (wily) yet.

**Comment by slyness** [comment]
I digress. There is actually no support at all for 15.04 (vivid) in the pgdg apt repositories. There is, however, support for 15.10 (wily) and we do have non-LTS supported distributions in that list already so I will add Wily Werewolf to the list after all. 

Since there is not repository available at that source for 15.04 you will never be able to use the apt_pgdg_postgresql recipe on that Ubuntu release. You will have to just use the server recipe and set apt repo to false and choose the particular postgresql package supported in the native Ubuntu repositories. 

Pull Request #345 will resolve this issue and is slated for the next cookbook release. It has been merged to develop currently.

**Comment by davejacobs** [comment]
Awesome, thanks!
